.github/workflows/android-ci.yml

```diff
@@ -9,18 +9,22 @@ on:
 jobs:
   build:
 
-    runs-on: macos-10.15
+    runs-on: macos-11
 
     steps:
     - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
     - name: Install system dependencies
       run: brew install boost ninja
     - name: Install Android Dependencies
       run: |
-        $ANDROID_HOME/tools/bin/sdkmanager --verbose "cmake;3.10.2.4988404" "ndk;21.2.6472646"
-        $ANDROID_HOME/tools/bin/sdkmanager "system-images;android-26;google_apis;x86"
+        $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --verbose "cmake;3.18.1" "ndk;23.1.7779620"
```

Comment:
should we have those commands specified in the `tools` or `make install-android-dependencies` ?

Reply:
+1